### PR TITLE
Tanya,miners,terrain and misc

### PR DESCRIPTION
**Pull request by tomsons26**
Add new clock anim,add Tanya,tweak terrain and speed defaults,make miners find ore, fix refin bibs,tweak misc stuff,move spysat to proper tab

### CODE REVIEW

File: weapons.yaml [hunk 1]
```diff
@@ -291,3 +291,29 @@ opcoilbolt: # when overcharged by 3 Tesla Troopers
 	Warhead@3EffWater: CreateEffect
 		Explosion: water_piffs
 		ValidImpactTypes: Water
+	
+DoublePistols:
+	ReloadDelay: 5
+	Range: 6c0
+	Report: itanatta.wav itanattb.wav
```
Comment (abcdefg30) [inline weapons.yaml]: You miss a comma here.
(It should be `Report: itanatta.wav, itanattb.wav`.)

**Comment by Phrohdoh** [comment]
These values need to be checked to make sure they match vanilla.

File: rules/infantry.yaml [hunk 2]
```diff
@@ -133,3 +133,40 @@ e2:
...
+		Prerequisites: ~gatech, ~gapile
+		BuildLimit: 1
+	Valued:
+		Cost: 1000
+	Tooltip:
+		Name: Tanya
+		Description: Elite commando infantry, armed with \ndual pistols and C4.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles\n  Special Ability: Destroy Building with C4\n\nMaximum 1 can be trained
+	Selectable:
+		Voice: TanyaVoice
+		Bounds: 12,17,0,-9
```
Comment (abcdefg30) [inline rules/infantry.yaml]: What about something like `Bounds: 20, 30, 0, -11` here?

**Comment by Phrohdoh** [comment]
Looks good to me. :+1:

**Pull request merged**